libs/sdk-ui-tests-e2e/cypress/fixtures/dashboardInfosForExport.json

```diff
@@ -0,0 +1,46 @@
+{
+    "validInsightsForTigerPDFExport": [
+        {
+            "dashboardTitle": "KD with compares",
+            "dashboardURL": "dashboard/compare",
+            "fileName": "KD with compares.pdf",
+            "contents": "Region\nEast CoastWest Coast\nStage Name\n\nSum of Amount: Stage Name (5)Sum of Amount: Stage Name (5)\nInterest112,261,028.81658,375,577.02\nDiscovery81,789,169.68181,439,545.32\nShort List–219,302,451.65\nAD with compares\nKD with compares"
```

Comment:
What does "\n" mean in the contents?

Reply:
It's just blank space. But it has nothing to do with the content of the pdf file. I removed it.